egs/swc/s5b/local/chain/multi_condition/run_tdnn_lstm.sh

```diff
@@ -0,0 +1 @@
+tuning/run_tdnn_lstm_1a.sh
```

Comment:
There are too many: 'local/chain/tuning/run_tdnn_lstm_*.sh' scripts. I would include there single script, which provides the best results (Please ask Dan or Yenda, which one it is).